# Cálculo del periodo de traslación de Venus.
Ejercicio de físca para practicar.

## Dibujo del problema.
![](https://thumbs.gfycat.com/EnchantingPositiveGermanshepherd-size_restricted.gif)

## Resolución del ejercicio.
1.- Cargar los paquetes de Sympy necesarios.

In [11]:
#Carga de objetos directos de Sympy
from sympy import solve, symbols, pi, Eq 
#Carga de unidades del S.I.
from sympy.physics.units import meter, kilogram, second, kelvin, newton, day 
#Carga de función fara convertir
from sympy.physics.units import convert_to 
#Carga  de la constante de gravitación universal
from sympy.physics.units import gravitational_constant as G  


La incógnita del problema es $T$:

In [4]:
T = symbols("T")

Datos conocidos:
* La masa del Sol ($M_\odot$) y,
* el semieje mayor de Venus (a)

In [5]:
a = 108208000e3 * meter
M0 = 1.9891e30 * kilogram

Usamos la ecuación de Kepler que relaciona los elementos anteriores:
$$
\frac{T^2}{a^3} = \frac{4\ \pi^2}{G\ M}
$$

Redactado en Sympy:

In [6]:
eq_kepler = Eq(T**2 / a**3, 4*pi**2 / G / M0)


Resolvemos la equación anterior:

In [13]:
solución = solve(eq_kepler, T)[1]
convert_to(solución, [day]).n(2)


2.2e+2*day

El resultado de aplicación de la formula de Kepler para obtener el periodo de traslación el planeta Venus, arroja un tiempo $T$ en días terrestres de $2.2\dot\ 10^2$ días.